redis/acknowledge.lua

```diff
@@ -1,8 +1,7 @@
 local zset_key = KEYS[1]
 local processed_key = KEYS[2]
 
-local worker_id = ARGV[1]
-local test = ARGV[2]
+local test = ARGV[1]
```

Comment:
That's my bad, it was an unused variable and I noticed it while I was implementing this PR.